src/pages/networks/forms/NetworkAcls.tsx

```diff
@@ -69,6 +70,24 @@ const NetworkAcls: FC<Props> = ({ formik, project }) => {
             id={networlAclSelectorId}
           />
         )}
+        {!formik.values.readOnly && (
+          <NetworkDefaultACLSelector
+            key={"NetworkDefaultACLSelector"}
+            id={""}
```

Comment:
```suggestion
            id={""}
```
We probably don't need a key and should have a valid id.